### ISSUE TITLE
Why does this plugin need to care about the base glyph’s category?

### ISSUE THREAD

**Issue by lianghai** [mekkablue/ShowMarkPreview#16]
Why do we need to keep opting in categories like this?

- https://github.com/mekkablue/ShowMarkPreview/issues/6
- https://github.com/mekkablue/ShowMarkPreview/issues/11

Why do we need to first define a glyph to be Letter or some other category in order to preview anchoring? Why isn’t the existence of a base-mark anchor pair enough?

**Comment by khaledhosny** [comment]
I came here to report the same issue. The plugin often fails to show preview on some glyphs, and I didn’t understand why until today when I was checking the code for some other issue and noticed the check for glyph category. For example, the plugin will not show mark preview on space glyph. Instead of adding another exception, I think the plugin should not check glyph category at all.

**Comment by lianghai** [comment]
@khaledhosny, I’ve been trying to migrate to https://github.com/simoncozens/ShowAnchorCloud. Glad you’ve been contributing to that plugin!

**Comment by khaledhosny** [comment]
Yes, with the latest changes ShowAnchorCloud plugin fits all my needs. This plugin has a different use cases though, e.g. showing the same marks over a number of glyphs at the same time and it would be more useful of this artificial limitation was removed.

**Comment by lianghai** [comment]
It’s probably easier to give ShowAnchorCloud the ability to only preview marks in the same preview line, than to change this plugin’s behavior (and improve its performance, correctly handle mark-to-mark attachment, …).

**Comment by mekkablue** [comment]
Sorry, missed the conversation. I do not remember exactly anymore, but it keep in mind it was originally built for Southeast Asian scripts, and filtering for the category made sense for that context.

**Comment by lianghai** [comment]
That sounds like a miscommunication when this plugin was originally developed. What’s even the value of disallowing the preview for certain base glyphs that the designer has intentionally placed a base anchor on?

**Comment by mekkablue** [comment]
As I said, I do not remember exactly. I needed to differentiate between mark and non-mark, and when I did change something, the existing users (SEA script designers) complained. 

I consider this issue closed because there is another plug-in that does what you need.
